src/featbin/wav-reverberate.cc

```diff
@@ -107,23 +137,36 @@ int main(int argc, char *argv[]) {
         "room-impulse response (rir_matrix) and additive noise distortions\n"
         "(specified by corresponding files).\n"
         "Usage:  wav-reverberate [options...] <wav-in-rxfilename> "
-        "<rir-rxfilename> <wav-out-wxfilename>\n"
+        "<wav-out-wxfilename>\n"
         "e.g.\n"
-        "wav-reverberate --noise-file=noise.wav \\\n"
-        "  input.wav rir.wav output.wav\n";
+        "wav-reverberate --duration=t --impulse-response=rir.wav "
+        "--additive-signals='noise1.wav,noise2.wav' --snrs='snr1,snr2' "
```

Comment:
The idea of an example (the bit after the 'e.g.') is to have an *actual* command line that could possibly work-- with concrete values.  So replace 't' and 'snr1,snr2' with actual values.  If you have made the same mistake elsewhere, please fix it.